doc/classes/Control.xml

```diff
@@ -565,10 +565,12 @@
 			</description>
 		</method>
 		<method name="grab_focus">
-			<return type="void" />
+			<return type="bool" />
+			<param index="0" name="auto_set_viewport_active_focus_layer" type="bool" default="false" />
 			<description>
-				Steal the focus from another control and become the focused control (see [member focus_mode]).
-				[b]Note:[/b] Using this method together with [method Callable.call_deferred] makes it more reliable, especially when called inside [method Node._ready].
+				Steal the focus from another control and become the focused control (see [member focus_mode]). If [param auto_set_viewport_active_focus_layer] is [code]true[/code], also sets the focused layer to the layer this control belongs to.
+				Returns [code]true[/code] if it was able to grab focus. Returns [code]false[/code] if the control doesn't get focused (see [member focus_mode]).
+				[b]Note[/b]: Using this method together with [method Callable.call_deferred] makes it more reliable, especially when called inside [method Node._ready].
```

Comment:
This is a lovely amount of detail and it's a shame it's locked behind this contested PR. Should be a separate PR outright.